rdkit/Chem/AtomPairs/Torsions.py

```diff
@@ -93,71 +93,70 @@ def ExplainPathScore(score, size=4):
 
   >>> from rdkit import Chem
   >>> m = Chem.MolFromSmiles('C=CC')
-  >>> score=pyScorePath(m,(0,1,2),3)
-  >>> ExplainPathScore(score,3)
+  >>> score=pyScorePath(m, (0, 1, 2), 3)
+  >>> ExplainPathScore(score, 3)
   (('C', 1, 0), ('C', 2, 1), ('C', 1, 1))
 
   Again, it's order independent:
 
-  >>> score=pyScorePath(m,(2,1,0),3)
-  >>> ExplainPathScore(score,3)
+  >>> score=pyScorePath(m, (2, 1, 0), 3)
+  >>> ExplainPathScore(score, 3)
   (('C', 1, 0), ('C', 2, 1), ('C', 1, 1))
 
   >>> m = Chem.MolFromSmiles('C=CO')
-  >>> score=pyScorePath(m,(0,1,2),3)
-  >>> ExplainPathScore(score,3)
+  >>> score=pyScorePath(m, (0, 1, 2), 3)
+  >>> ExplainPathScore(score, 3)
   (('C', 1, 1), ('C', 2, 1), ('O', 1, 0))
 
   >>> m = Chem.MolFromSmiles('OC=CO')
-  >>> score=pyScorePath(m,(0,1,2,3),4)
-  >>> ExplainPathScore(score,4)
+  >>> score=pyScorePath(m, (0, 1, 2, 3), 4)
+  >>> ExplainPathScore(score, 4)
   (('O', 1, 0), ('C', 2, 1), ('C', 2, 1), ('O', 1, 0))
 
   >>> m = Chem.MolFromSmiles('CC=CO')
-  >>> score=pyScorePath(m,(0,1,2,3),4)
-  >>> ExplainPathScore(score,4)
+  >>> score=pyScorePath(m, (0, 1, 2, 3), 4)
+  >>> ExplainPathScore(score, 4)
   (('C', 1, 0), ('C', 2, 1), ('C', 2, 1), ('O', 1, 0))
 
 
   >>> m = Chem.MolFromSmiles('C=CC(=O)O')
-  >>> score=pyScorePath(m,(0,1,2,3),4)
-  >>> ExplainPathScore(score,4)
+  >>> score=pyScorePath(m, (0, 1, 2, 3), 4)
+  >>> ExplainPathScore(score, 4)
   (('C', 1, 1), ('C', 2, 1), ('C', 3, 1), ('O', 1, 1))
-  >>> score=pyScorePath(m,(0,1,2,4),4)
-  >>> ExplainPathScore(score,4)
+  >>> score=pyScorePath(m, (0, 1, 2, 4), 4)
+  >>> ExplainPathScore(score, 4)
   (('C', 1, 1), ('C', 2, 1), ('C', 3, 1), ('O', 1, 0))
 
 
   >>> m = Chem.MolFromSmiles('OOOO')
-  >>> score=pyScorePath(m,(0,1,2),3)
-  >>> ExplainPathScore(score,3)
+  >>> score=pyScorePath(m, (0, 1, 2), 3)
+  >>> ExplainPathScore(score, 3)
   (('O', 1, 0), ('O', 2, 0), ('O', 2, 0))
-  >>> score=pyScorePath(m,(0,1,2,3),4)
-  >>> ExplainPathScore(score,4)
+  >>> score=pyScorePath(m, (0, 1, 2, 3), 4)
+  >>> ExplainPathScore(score, 4)
   (('O', 1, 0), ('O', 2, 0), ('O', 2, 0), ('O', 1, 0))
 
   """
   codeMask = (1 << rdMolDescriptors.AtomPairsParameters.codeSize) - 1
   res = [None] * size
   for i in range(size):
-    if i == 0 or i == (size - 1):
+    if i in (0, size - 1):
```

Comment:
I'm not sure if I find this easier to read. Intuitively, I look at this and interpret it as an interval.

Reply:
Yeah, I also don't find this form easier to read.

Not that it's relevant here, but as an FYI this form is also considerably slower:
```
In [1]: def f1(size,nreps):
   ...:     for i in range(nreps):
   ...:         accum = 0
   ...:         for j in range(size):
   ...:             if j in (0,size-1):
   ...:                 accum += j
   ...: 

In [2]: def f2(size,nreps):
   ...:     for i in range(nreps):
   ...:         accum = 0
   ...:         for j in range(size):
   ...:             if j ==0 or j == size-1:
   ...:                 accum += j
   ...: 
   ...: 

In [3]: %timeit f1(100,1000)
4.86 ms ± 41.1 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [4]: %timeit f2(100,1000)
3.72 ms ± 11.6 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

```

Reply:
@IchiruTake : do you have a reason for preferring the `in` form here instead of using the two equality checks?